plugins/normalizeColorCase.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+exports.type = 'perItem';
+
+exports.active = false;
```

Comment:
@TrySound this plugin should be enabled by default in the future IMHO since it should improve gzip compression